fe/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -418,7 +414,19 @@ public void setDisableColocateJoin(boolean disableColocateJoin) {
         this.disableColocateJoin = disableColocateJoin;
     }
 
-    // Serialize to thrift object
+    public int getParallelExecInstanceNum() {
+        return parallelExecInstanceNum;
+    }
+
+    public void setParallelExecInstanceNum(int parallelExecInstanceNum) {
+        if (parallelExecInstanceNum < MIN_EXEC_INSTANCE_NUM) {
```

Comment:
ParallelExecInstanceNum should have a upper boundary，such as 32, 64 or 128.

Reply:
OK